src/main/java/org/eclipse/golo/doc/FunctionDocumentation.java

```diff
@@ -170,25 +170,50 @@ public FunctionDocumentation parent(DocumentationElement p) {
 
   @Override
   public int compareTo(DocumentationElement other) {
-    if (this == other) { return 0; }
-    if (null == other) { return 1; }
+    if (this == other) {
+      return 0;
+    }
+    if (null == other) {
+      return 1;
+    }
     if (other instanceof FunctionDocumentation) {
       FunctionDocumentation o = (FunctionDocumentation) other;
       int c = name.compareToIgnoreCase(o.name);
       if (c == 0) {
         c = arity() < o.arity() ? -1 : 1;
-        if (c == 0) {
-          c = varargs && !o.varargs() ? 1
-              : varargs && o.varargs() ? 0
-              : -1;
-          if (c == 0) {
-            c = parent.compareTo(o.parent);
-          }
-        }
       }
```

Comment:
My point is that is should not be dead code :smile:
I rather write `c = Integer.compare(arity(), o.artity())` and thus the `c == 0` case is no more dead code. But I may be wrong. However, it's only used to sort the functions in the documentations, so no big deal. (nb: it seems that I'm the author of this infamy :sweat:)